packages/axios-extension/package.json

```diff
@@ -35,7 +35,8 @@
         "open": "7.0.3",
         "qs": "6.7.0",
         "xpath": "0.0.32",
-        "@xmldom/xmldom": "0.8.2"
+        "@xmldom/xmldom": "0.8.2",
+        "i18next": "21.6.11"
```

Comment:
We don't need translatable texts here, this is a low-level library, please remove `i18next` and keep the texts inline.

Reply:
Removed i18next translation